legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-postgres/src/test/resources/data/unitemporal-incremental-milestoning/input/batch_id_based/without_delete_ind/staging_data_pass3.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
we dont need this file

Reply:
Also it seems like the data files for bitemporal snapshot are included, we can delete them